services/vision/mlvision/ml_model_test.go

```diff
@@ -517,6 +519,40 @@ func TestMultipleClassifiersOneModel(t *testing.T) {
 	wg.Wait()
 }
 
+func TestRegistrationWithDefaultCamera(t *testing.T) {
+	ctx := context.Background()
+	out := mockEffDetModel("myMLModel", "")
+	modelName := out.Name()
+	cameraName := camera.Named("test")
+	modelCfg := MLModelConfig{ModelName: modelName.Name}
+
+	r := &inject.Robot{}
+	r.ResourceByNameFunc = func(name resource.Name) (resource.Resource, error) {
+		switch name {
+		case modelName:
+			return out, nil
+		case cameraName:
+			return inject.NewCamera(cameraName.Name), nil
+		default:
+			return nil, resource.NewNotFoundError(name)
+		}
+	}
+
+	service, err := registerMLModelVisionService(ctx, modelName, &modelCfg, r, logging.NewLogger("benchmark"))
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, service, test.ShouldNotBeNil)
```

Comment:
Just for the mlvision service, can you write a test that calls *FromCamera after it registers a no-default camera service.  Literally just try one of them like service.DetectionsFromCamera(nil input) and make sure that you get that error b/c there's no camera.